tools/proteore_get_unique_peptide_srm_method/proteore_get_unique_peptide_SRM-MRM_method.xml

```diff
@@ -0,0 +1,167 @@
+<tool id="proteore_get_unique_peptide_srm_method" name="get unique peptide srm-mrm method" version="2021.06.14">
+    <description>[Human SRM Atlas]</description>
+    <requirements>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+
+        python '$__tool_directory__/get_unique_srm.py'
+        
+        #if $input.ids == 'text'
+            --input='$input.txt'
+            --input_type='list'
+        #else
+            --input='$input.input_file'
+            --column_number='$input.ncol'
+            --header='$input.header'
+            --input_type='file'
+        #end if
+
+        --features='$protein_features'
+        --output='$output'
+        
+        #if "SRM_atlas" in str($srm_file).split("/")
+            --ref_file='$srm_file'
+        #else 
+            --ref_file='$__tool_directory__/$srm_file'
+        #end if 
+
+    ]]></command>
+    <inputs>
+    <conditional name="input" >
+        <param name="ids" type="select" label="Enter IDs (Uniprot Accession number, e.g. P04746)" help="Copy/paste or from a file (e.g. table)" >
+            <option value="text">Copy/paste your Uniprot-AC identifiers</option>
+            <option value="file" selected="true">Input file containing Uniprot-AC identifiers</option>
+        </param>
+        <when value="text" >
+            <param name="txt" type="text" label="Copy/paste IDs" help='IDs must be separated by tab, space or carriage return into the form field, for example: P31946 P62258' >
+                <sanitizer invalid_char="">
+                    <valid initial="string.printable">
+                        <remove value="&apos;"/>
+                    </valid>
+                    <mapping initial="none">
+                        <add source="&apos;" target="__sq__"/>
+                        <add source="&#x20;" target=""/>
+                        <add source="&#xA;" target=""/>
+                        <add source="&#xD;" target=""/>
+                        <add source="&#x9;" target=""/>
+                    </mapping>
+                </sanitizer>
+            </param>
+        </when>
+        <when value="file" >
+            <param name="input_file" type="data" format="txt,tabular" label="Select your file" help="" />
+            <param name="header" type="boolean" checked="true" truevalue="true" falsevalue="false" label="Does file contain header?" />
+            <param name="ncol" type="text" value="c1" label="Column number of IDs" help='For example, fill in "c1" if it is the first column, "c2" if it is the second column and so on'>
+                <validator type="regex" message="Please enter a column number, for example: 'c1' for the first column">[c]{0,1}[0-9]+</validator>
+            </param>
+        </when>
+    </conditional>
+    <param name="srm_file" type="select" label="Release" >
+        <options from_data_table="proteore_human_srm_atlas">
+            <filter type="sort_by" column="0"/>
```

Comment:
Do we realy want to sort for the ID column? E.g. `9979839599` ... what is this number? Should we sort for column 1?